{
 "about": {
  "channels": [
   "https://conda.anaconda.org/conda-forge/osx-64",
   "https://conda.anaconda.org/conda-forge/noarch",
   "https://repo.continuum.io/pkgs/free/osx-64",
   "https://repo.continuum.io/pkgs/free/noarch",
   "https://repo.continuum.io/pkgs/pro/osx-64",
   "https://repo.continuum.io/pkgs/pro/noarch"
  ],
  "conda_build_version": "2.1.10",
  "conda_env_version": "4.2.13",
  "conda_private": false,
  "conda_version": "4.2.13",
  "dev_url": "http://github.com/datadotworld/data.world-py",
  "env_vars": {
   "CIO_TEST": "<not set>",
   "CONDA_DEFAULT_ENV": "root",
   "CONDA_ENVS_PATH": "<not set>",
   "DYLD_LIBRARY_PATH": "<not set>",
   "PATH": "/Users/travis/miniconda3/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p643/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin",
   "PYTHONHOME": "<not set>",
   "PYTHONPATH": "<not set>"
  },
  "home": "http://github.com/datadotworld/data.world-py",
  "license": "Apache 2.0",
  "license_family": "Apache",
  "offline": false,
  "root_pkgs": [
   "conda-forge::anaconda-client-1.6.3-py36_0",
   "asn1crypto-0.22.0-py36_0",
   "conda-forge::beautifulsoup4-4.5.3-py36_0",
   "cffi-1.10.0-py36_0",
   "conda-forge::chardet-3.0.2-py36_1",
   "conda-forge::clyent-1.2.1-py36_0",
   "conda-forge::conda-4.2.13-py36_0",
   "conda-forge::conda-build-2.1.10-py36_0",
   "conda-forge::conda-env-2.6.0-0",
   "conda-forge::conda-forge-build-setup-4.4.4-0",
   "conda-forge::conda-verify-2.0.0-py36_0",
   "cryptography-1.8.1-py36_0",
   "conda-forge::filelock-2.0.6-py36_0",
   "idna-2.5-py36_0",
   "conda-forge::jinja2-2.9.5-py36_0",
   "conda-forge::markupsafe-1.0-py36_0",
   "openssl-1.0.2l-0",
   "packaging-16.8-py36_0",
   "pip-9.0.1-py36_1",
   "conda-forge::pkginfo-1.2.1-py36_0",
   "pycosat-0.6.2-py36_0",
   "pycparser-2.17-py36_0",
   "conda-forge::pycrypto-2.6.1-py36_1",
   "pyopenssl-17.0.0-py36_0",
   "pyparsing-2.1.4-py36_0",
   "python-3.6.1-2",
   "conda-forge::python-dateutil-2.6.0-py36_0",
   "conda-forge::pytz-2017.2-py36_0",
   "conda-forge::pyyaml-3.12-py36_1",
   "readline-6.2-2",
   "requests-2.14.2-py36_0",
   "ruamel_yaml-0.11.14-py36_1",
   "setuptools-27.2.0-py36_0",
   "six-1.10.0-py36_0",
   "sqlite-3.13.0-0",
   "tk-8.5.18-0",
   "wheel-0.29.0-py36_0",
   "xz-5.2.2-1",
   "yaml-0.1.6-0",
   "zlib-1.2.8-3"
  ],
  "summary": "Python library for data.world"
 },
 "conda_build_config": {},
 "files": [
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/error_message.py",
  "lib/python3.5/site-packages/datadotworld/datadotworld.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_create_or_update_request.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/apis/__init__.py",
  "lib/python3.5/site-packages/datadotworld/models/table_schema.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_source_summary_response.py",
  "lib/python3.5/site-packages/datadotworld/models/__init__.py",
  "lib/python3.5/site-packages/datadotworld/client/__init__.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_source_create_or_update_request.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_source_create_request.py",
  "lib/python3.5/site-packages/datadotworld/config.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_summary_response.py",
  "lib/python3.5/site-packages/datadotworld/client/api.py",
  "lib/python3.5/site-packages/datadotworld/cli.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/dataset_put_request.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_create_request.py",
  "lib/python3.5/site-packages/datadotworld/models/dataset.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/rest.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/__init__.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/configuration.py",
  "lib/python3.5/site-packages/datadotworld/__init__.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/dataset_create_request.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/api_client.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/file_batch_update_request.py",
  "lib/python3.5/site-packages/datadotworld/models/query.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/dataset_patch_request.py",
  "lib/python3.5/site-packages/datadotworld-1.2.5-py3.5.egg-info/PKG-INFO",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/dataset_summary_response.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/apis/datasets_api.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/__init__.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/apis/uploads_api.py",
  "bin/dw",
  "lib/python3.5/site-packages/datadotworld/util.py",
  "lib/python3.5/site-packages/datadotworld/client/_swagger/models/success_message.py"
 ],
 "index": {
  "arch": "x86_64",
  "build": "py35_0",
  "build_number": 0,
  "depends": [
   "certifi >=2017.04.17",
   "click >=6.0,<7.0a",
   "datapackage-py >=0.8.8,<1.0a",
   "jsontableschema >=0.10.0,<1.0a",
   "pandas <1.0a",
   "python 3.5*",
   "python-dateutil >=2.6.0,<3.0a",
   "requests >=2.0.0,<3.0a",
   "six >=1.5.0,<2.0a",
   "tabulator-py <2.0a",
   "urllib3 >=1.15,<2.0a"
  ],
  "license": "Apache 2.0",
  "license_family": "Apache",
  "name": "datadotworld-py",
  "platform": "osx",
  "subdir": "osx-64",
  "version": "1.2.5"
 },
 "metadata_version": 1,
 "name": "datadotworld-py",
 "raw_recipe": "{% set conda_name = \"datadotworld-py\" %}\n{% set name = \"datadotworld\" %}\n{% set version = \"1.2.5\" %}\n{% set bundle = \"tar.gz\" %}\n{% set hash_type = \"sha256\" %}\n{% set hash = \"484eaed3e44563274634daf67d134268668fa08fd4b7e72b1a151fe24ca5f531\" %}\n{% set build = 0 %}\n\npackage:\n  name: {{ conda_name }}\n  version: {{ version }}\n\nsource:\n  fn: {{ name }}-{{ version }}.{{ bundle }}\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}\n  {{ hash_type }}: {{ hash }}\n\nbuild:\n  entry_points:\n    - dw=datadotworld.cli:cli\n\n  number: {{ build }}\n  script: python setup.py install --single-version-externally-managed --record=record.txt\n\nrequirements:\n  build:\n    - python\n    - setuptools\n    - pytest-runner >=2.11,<3.0a\n\n  run:\n    - python\n    - certifi >=2017.04.17\n    - click >=6.0,<7.0a\n    - configparser >=3.5.0,<4.0a  # [py<34]\n    - datapackage-py >=0.8.8,<1.0a\n    - jsontableschema >=0.10.0,<1.0a\n    - python-dateutil >=2.6.0,<3.0a\n    - requests >=2.0.0,<3.0a\n    - six >=1.5.0,<2.0a\n    - tabulator-py <2.0a\n    - urllib3 >=1.15,<2.0a\n    - pandas <1.0a\n\ntest:\n  imports:\n    - datadotworld\n    - datadotworld.client\n    - datadotworld.client._swagger\n    - datadotworld.client._swagger.apis\n    - datadotworld.client._swagger.models\n    - datadotworld.models\n\n  commands:\n    - dw --help\n\nabout:\n  home: http://github.com/datadotworld/data.world-py\n  license_file: LICENSE.txt\n  license: Apache 2.0\n  license_family: Apache\n  summary: 'Python library for data.world'\n  dev_url: http://github.com/datadotworld/data.world-py\n\nextra:\n  recipe-maintainers:\n    - pmlandwehr\n",
 "rendered_recipe": {
  "about": {
   "dev_url": "http://github.com/datadotworld/data.world-py",
   "home": "http://github.com/datadotworld/data.world-py",
   "license": "Apache 2.0",
   "license_family": "Apache",
   "license_file": "LICENSE.txt",
   "summary": "Python library for data.world"
  },
  "build": {
   "entry_points": [
    "dw=datadotworld.cli:cli"
   ],
   "noarch": "",
   "noarch_python": false,
   "number": "0",
   "script": "python setup.py install --single-version-externally-managed --record=record.txt",
   "string": "py35_0"
  },
  "extra": {
   "final": true,
   "recipe-maintainers": [
    "pmlandwehr"
   ]
  },
  "package": {
   "name": "datadotworld-py",
   "version": "1.2.5"
  },
  "requirements": {
   "build": [
    "ca-certificates 2017.4.17 0",
    "certifi 2017.4.17 py35_0",
    "ncurses 5.9 10",
    "openssl 1.0.2k 0",
    "py 1.4.33 py35_0",
    "pytest 3.1.1 py35_0",
    "pytest-runner 2.11.1 py35_0",
    "python 3.5.3 3",
    "readline 6.2 0",
    "setuptools 33.1.1 py35_0",
    "sqlite 3.13.0 1",
    "tk 8.5.19 1",
    "xz 5.2.2 0",
    "zlib 1.2.11 0"
   ],
   "run": [
    "python",
    "certifi >=2017.04.17",
    "click >=6.0,<7.0a",
    "datapackage-py >=0.8.8,<1.0a",
    "jsontableschema >=0.10.0,<1.0a",
    "python-dateutil >=2.6.0,<3.0a",
    "requests >=2.0.0,<3.0a",
    "six >=1.5.0,<2.0a",
    "tabulator-py <2.0a",
    "urllib3 >=1.15,<2.0a",
    "pandas <1.0a"
   ]
  },
  "source": {
   "fn": "datadotworld-1.2.5.tar.gz",
   "sha256": "484eaed3e44563274634daf67d134268668fa08fd4b7e72b1a151fe24ca5f531",
   "url": "https://pypi.io/packages/source/d/datadotworld/datadotworld-1.2.5.tar.gz"
  },
  "test": {
   "commands": [
    "dw --help"
   ],
   "imports": [
    "datadotworld",
    "datadotworld.client",
    "datadotworld.client._swagger",
    "datadotworld.client._swagger.apis",
    "datadotworld.client._swagger.models",
    "datadotworld.models"
   ]
  }
 },
 "version": "1.2.5"
}